src/Dependencies/Microsoft.DiaSymReader/ISymUnmanagedAsyncMethod.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.DiaSymReader
     public interface ISymUnmanagedAsyncMethod
     {
         [PreserveSig]
-        int IsAsyncMethod(out bool value);
+        int IsAsyncMethod([MarshalAs(UnmanagedType.Bool)]out bool value);
```

Comment:
Note that, if you do remove the [PreserveSig], for this case you would have to annotate the return type with ```[return: MarshalAs(UnmanagedType.Bool)]```